config_splunk.go

```diff
@@ -24,20 +24,7 @@ func (c *SplunkConfig) Get(env Environment) error {
 		if err != mongo.ErrNoDocuments {
 			return errors.Wrapf(err, "retrieving section '%s'", c.SectionId())
 		}
-		// TODO EVG-17353: remove retrieving settings from global
-		globalConfig := coll.FindOne(ctx, byId(ConfigDocID))
-		if err := globalConfig.Err(); err != nil {
-			if err != mongo.ErrNoDocuments {
-				return errors.Wrap(err, "retrieving global settings")
-			}
-			c = &SplunkConfig{}
-			return nil
-		}
-		s := Settings{}
-		if err := globalConfig.Decode(&s); err != nil {
-			return errors.Wrap(err, "decoding global config")
-		}
-		c.SplunkConnectionInfo = s.Splunk.SplunkConnectionInfo
+		c = &SplunkConfig{}
```

Comment:
Do we need this considering that c is passed in? I'm assuming c is already defined since it's being accessed above?

Reply:
i think this is to make sure that the config is empty if we do find an empty document like in here? 
https://github.com/evergreen-ci/evergreen/blob/264ae58b71ecdef7a5f1c9e9bc5020288033f8cd/config_slack.go#L28https://github.com/evergreen-ci/evergreen/blob/264ae58b71ecdef7a5f1c9e9bc5020288033f8cd/config_slack.go#L28

Reply:
ah that makes sense. Do we want to maybe flip the if just to be consistent with the function you linked?

Reply:
i think @ablack12 preferred it written this way but i wasnt sure why

Reply:
I wanted it that way in the original PR so that the bulk of the code wouldn't have to be nested, since that isn't as readable (I did mention this in the original PR: https://github.com/evergreen-ci/evergreen/pull/5812#discussion_r929227347) . Now that most of the code is being deleted, this isn't relevant.

Reply:
updated!